CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `quint run` returns a non-zero exit code on invariant violation and errors (#793)
 - Support for imports from files: `import ... from <filename>` (#800)
 - suppress comments in REPL (#806)
+- Support for imports from files: `import ... from <filename>` (#800)
```

Comment:
is this picked up from an earlier commit? maybe this PR needs a rebase?

Reply:
ah weird! I must have messed this up in merging. I tried to avoid rebasing because I'm not sure how it will behave since I created this branch on top of Thomas' and then merged Igor's WIP branch on it before starting to work. Perhaps it will be easy to rebase if I just drop all of Igor's commits, as the ics20 files shouldn't have a lot of conflicts. I'll try. If it's to messy, I'll just remove this and squash merge :stuck_out_tongue:

Reply:
Rebasing seems to have worked fine!